caffe2/CMakeLists.txt

```diff
@@ -1860,4 +1874,4 @@ if(BUILD_PYTHON)
   # Caffe2 proto files
   install(DIRECTORY ${CMAKE_BINARY_DIR}/caffe2 DESTINATION ${PYTHON_LIB_REL_PATH}
           FILES_MATCHING PATTERN "*.py")
-endif()
+endif()
```

Comment:
Please keep the newline character after endif

Reply:
I got all mixed up `ghstack`-ing things around and a bunch of stuff got reverted, it should all be fixed now